<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2102</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #e6f7f5; /* light greenish background */
    color: #003366; /* dark blue text */
    font-size: 14px;
}

QPushButton {
    background-color: #4CAF50; /* green button */
    color: white;
    border: none;
    padding: 6px 12px;
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #45a049; /* darker green on hover */
}

QLineEdit, QTextEdit {
    background-color: #ffffff;
    border: 2px solid #3399ff; /* blue border */
    border-radius: 4px;
    padding: 4px;
}

QLineEdit:focus, QTextEdit:focus {
    border: 2px solid #0066cc; /* darker blue when focused */
}

QLabel {
    color: #004d66; /* teal text */
    font-weight: bold;
}
QTabWidget::pane {
    border: 2px solid #3399ff; /* blue border */
    border-radius: 6px;
    background: #e6f7f5; /* light greenish background */
    padding: 4px;
}

QTabBar::tab {
    background: #4CAF50; /* green tabs */
    color: white;
    border: 1px solid #3399ff; /* blue border */
    border-radius: 6px;
    padding: 6px 12px;
    margin-right: 2px;
}

QTabBar::tab:selected {
    background: #3399ff; /* blue when selected */
    color: white;
}

QTabBar::tab:hover {
    background: #45a049; /* darker green hover */
}
QListView {
    background-color: #ffffff;
    border: 2px solid #3399ff; /* blue border */
    border-radius: 6px;
    padding: 4px;
    show-decoration-selected: 1; /* highlight whole row */
}

QListView::item {
    padding: 6px;
    border-bottom: 1px solid #e6f7f5; /* subtle divider */
    color: #003366; /* dark blue text */
}

QListView::item:selected {
    background-color: #4CAF50; /* green selection */
    color: white;
    border: none;
}

QListView::item:hover {
    background-color: #cceeff; /* light blue hover */
    color: #004d66;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>130</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Trie Par Nom</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>311</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N Tel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenom">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_email">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_motdepass">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_telephon">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Poste</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_poste">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>270</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Salaire</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_salaire">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>320</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_cin2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_email2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N Tel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_tel2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenom2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_motdepass2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>370</y>
        <width>90</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_poste_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Salaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Poste</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_salaire_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>stat</string>
     </attribute>
     <widget class="QPushButton" name="refresh_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher_stat</string>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>281</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>790</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>30</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_13">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>310</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>220</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QTableView" name="tab_emp">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>601</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>790</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Recherche Par Cin</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2102</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
